corehq/ex-submodules/couchforms/tests/test_analytics.py

```diff
@@ -111,6 +111,8 @@ def test_get_form_analytics_metadata__app(self):
 
     @flaky_slow
     def test_get_exports_by_form(self):
+        # Call this twice, since the couchdb `update_after` to force couchdb to return the right results
+        get_exports_by_form(self.domain)
```

Comment:
Does calling it twice actually force it, or does it just take a bit more time, making the second call a bit more likely to return the correct results? In other words, is this possibly a flaky test?

Reply:
I think it actually forces it. This is also a flaky test, but it was before I added this change (and is annotated as such)